tests/ops/qubit/test_parametric_ops.py

```diff
@@ -1061,7 +1071,9 @@ def test_pcphase_jax(self, phi, dim, wires):
         mat2 = op.compute_matrix(*op.parameters, **op.hyperparameters)
 
         expected_mat = jnp.diag(
-            jnp.array([jnp.exp(1j * phi) if i < dim else 1.0 for i in range(2**num_wires)])
+            jnp.array(
```

Comment:
How this is computed? If computed by hand or it is a trivial solution, might add a comment to clarify.